config/Makefile.linux-mingw64

```diff
@@ -6,11 +6,11 @@
 
 NAME  := glew32
 HOST  := i686-w64-mingw32
-CC    := $(HOST)-gcc
-LD    := $(HOST)-ld
+CC    := $(HOST)-gcc -fno-builtin -fno-stack-protector
```

Comment:
I'd prefer these flags moved to _CFLAGS.EXTRA_ so that overriding CC and LD doesn't result in these being lost.